.mvn/modernizer/violations.xml

```diff
@@ -313,4 +313,16 @@
         <version>1.8</version>
         <comment>Prefer org.testng.annotations.AfterClass</comment>
     </violation>
+
+    <violation>
+        <name>com/fasterxml/jackson/core/JsonFactory."&lt;init&gt;":()V</name>
+        <version>1.8</version>
+        <comment>Use io.trino.plugin.base.util.JsonUtils.jsonFactory()</comment>
```

Comment:
squash "Ban direct usage of JsonFactory and JsonFactoryBuilder" with previous commit